enclave_manager/avalon_enclave_manager/singleton/singleton_enclave_manager.py

```diff
@@ -384,6 +145,9 @@ def main(args=None):
         logger.error(str(e))
         sys.exit(-1)
 
+    if options.workloads:
+        config["WorkerConfig"]["workloads"] = options.workloads
```

Comment:
Are we passing down these workload ids to C++ layer to match these ids with actual workload ids in registration? What if pass dummy workload id and submit a work order request against workload?

Reply:
The system can be bogged down using it but the workorder will not be processed anyhow. No this is restricted to python layer not percolated to CPP.